bonfire/bonfire.py

```diff
@@ -51,7 +54,12 @@
     merge_app_configs,
 )
 
+
 log = logging.getLogger(__name__)
+es_telemetry = logging.getLogger("elasticsearch")
```

Comment:
Right now if we import anything from `bonfire` it would set up this logger. So I think even during a unit test (for example), this logger is going to get set up. I am thinking we should move the setup of this into a function... like `def setup_telemetry` and maybe you have the `main_with_handler` func call `setup_telemetry()`?

Reply:
I moved the handler setup to `main_with_handler`. It does make some things a little more complicated. i.e. Grabbing the handler from the logger because the handler isn't in scope. Not a big deal though